### ISSUE TITLE
leaf nodes not rendered

### ISSUE THREAD

**Issue by evancarey** [evancarey/jsTreemap#1]
v0.4.0 In windows 7 and FF 14.0.1 leaf nodes are not rendered in example tm in index.html.  Geometry is created and can be seen via the highlighter as mouse moves across canvas.  FF console shows no errors.

**Comment by bgrohman** [comment]
Hey, I just forked your repo and re-tested on FF 14.01 in Windows 7. I can't reproduce the issue locally or on jstreemap.com now. Weird.

**Comment by evancarey** [comment]
I bet something was cached.  I've seen similar probs after I've made
updates to the datafile that holds the mouseover box content.

**Comment by bgrohman** [comment]
Yeah, could be. I cleared cache repeatedly, though, when it happened.
